README.md

```diff
@@ -201,35 +224,125 @@ the following in [.travis.yml](.travis.yml).
+automatically kick off a build.
 
 ### Installing Jenkins
 
 Before beginning, you'll need to set up your own installation of Jenkins.  Check out the 
-[Jenkins Installation Guide](https://jenkins.io/doc/book/installing/) for more details.  We recommend installing the 
-Jenkins suggested plugins.
+[Jenkins Installation Guide](https://jenkins.io/doc/book/installing/) for more details.  
+
+To help you get started, we have provided [PrepareJenkinsDockerContainer.sh](scripts/PrepareJenkinsDockerContainer.sh) 
+that you can run to start Jenkins inside of a Docker container.  The machine or container where Jenkins is running will 
+need to have [Docker Compose](https://docs.docker.com/compose/install/) and [Perl](https://www.perl.org/get.html) 
+installed. [PrepareJenkinsDockerContainer.sh](scripts/PrepareJenkinsDockerContainer.sh) will take care of installing 
+both for you. If you use [PrepareJenkinsDockerContainer.sh](scripts/PrepareJenkinsDockerContainer.sh), you should be 
+able to access Jenkins at [http://localhost:8080](http://localhost:8080).
+
+We recommend installing the Jenkins suggested plugins.
+
+### Storing Credentials in Jenkins
+
+The build you will create in Jenkins needs access to credentials and configurations.  We will store these using
+Jenkins Credentials.
+
+The instructions below will walk you through how to create the necessary credentials and configurations.  For more 
+information on how to store credentials in Jenkins, see 
+[Credentials Binding Plugin](https://wiki.jenkins.io/display/JENKINS/Credentials+Binding+Plugin).
+
+1. On the Jenkins dashboard, click **Credentials**.  (If you do not see the Credentials option, you may need to install the
+Credentials Binding Plugin.)
+1. Create new global credentials for the GitHub account that has access to the 
+[SugarCRM Unit Tests repo](https://github.com/sugarcrm/unit-tests).  
+   1. Kind: **Username with password**
+   1. Scope: **Global** 
+   1. Username: your GitHub username
+   1. Password: your GitHub password
+   1. ID: GITHUB_SUGARCRM_UNIT_TESTS
+   1. Description: GitHub account that has access to https://github.com/sugarcrm/unit-tests
+1. Create new global credentials for the account that has access to the 
+   [SugarCRM Developer Builds Space](https://community.sugarcrm.com/community/developer/developer-builds).  
+   1. Kind: **Username with password**
+   1. Scope: **Global** 
+   1. Username: your SugarCRM Developer Builds Space username
+   1. Password: your SugarCRM Developer Builds Space password
+   1. ID: SUGARCRM_DEVELOPER_BUILDS_SPACE
+   1. Description: Account that has access to the SugarCRM Developer Builds Space
+1. Create a new global secret for the path to where Sugar Docker will be stored on your host machine (not the path on 
+Jenkins).
+   1. Kind: **Secret text**
+   1. Scope: **Global** 
+   1. Secret: The path to where Sugar Docker will be stored on your host machine.  For example, if your Jenkins 
```

Comment:
I don't really understand this piece. It's a bit tricky for me. You have to enter this path here and also in the Jenkins build script further below in a couple different paths. Is there a way to just provide a root path and then you figure out all the necessary relative paths?

Reply:
I opened #52 to track this